frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -75,32 +53,31 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
       </EmptyState>
     );
   }
+  const totalHumanized = humanizeBinaryBytesWithoutB(totalUsage, 'Mi');
```

Comment:
i think this is not correct. `humanizeBinaryBytesWithoutB` arguments are `value`, `initialUnit`, `preferredUnit`. You are setting `initialValue` to `Mi` but it actually is `i`. Correct usage should be  `const totalHumanized = humanizeBinaryBytesWithoutB(totalUsage, null,  totalUsage ? null : 'Mi');` - you want `Mi` as preferred unit only if totalUsage is 0, and initialUnit is always a default one (bytes).

Reply:
thanks @rawagner ... tested with the mock data, works perfectly fine... (=

Reply:
Updated the PR...